processor/elasticinframetricsprocessor/internal/metadata/generated_status.go

```diff
@@ -24,7 +24,8 @@ import (
 )
 
 var (
-	Type = component.MustNewType("elasticinframetrics")
+	Type      = component.MustNewType("elasticinframetrics")
+	ScopeName = "github.com/elastic/opentelemetry-collector-components/processor/elasticinframetricsprocessor"
```

Comment:
Is this also a requirement of the version bump ?

Reply:
Not sure, that is generated file using mdatagen so I don't think it should matter much.